cmd/boost/deals.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+
+	bcli "github.com/filecoin-project/boost/cli"
+	"github.com/google/uuid"
+	"github.com/urfave/cli/v2"
+)
+
+var dealCmd = &cli.Command{
+	Name:  "deal",
+	Usage: "Manage Boost deals",
+	Subcommands: []*cli.Command{
+		makeOfflineDealWithData,
+	},
+}
+
+var makeOfflineDealWithData = &cli.Command{
+	Name:  "offline-deal-with-data",
```

Comment:
For simplicity maybe let's just call the command "offline-deal"

Reply:
Done.